### PR TITLE
Added enum for the `AuthStatus` and set the node by default on offline

### LINKED ISSUE #1461: [Bug] Node status is `None` at the server when node has never been online
**Describe the bug**
Should be `offline`. / `never-seen`

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**
Fixes #1461

### CODE REVIEW

**Review by wbaccinelli — APPROVED** [review]
Looks good, the code is also cleaner with the Enum.

File: vantage6-algorithm-tools/vantage6/algorithm/tools/mock_client.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import json
 import logging
+import pandas as pd
```
Comment (bartvanb) [inline vantage6-algorithm-tools/vantage6/algorithm/tools/mock_client.py:3]: Doesn't linting complain about putting this import here? As in https://github.com/PyCQA/flake8-import-order: "In general stdlib comes first, then 3rd party, then local packages, and that each group is individually alphabetized, however this depends on the style used. Flake8-Import-Order supports a number of [styles](https://github.com/PyCQA/flake8-import-order#styles) and is extensible allowing for [custom styles](https://github.com/PyCQA/flake8-import-order#extending-styles)."

File: vantage6-server/vantage6/server/websockets.py [hunk 2]
```diff
@@ -343,8 +344,8 @@ def on_ping(self) -> None:
         ping and sets them as online.
         """
         auth = db.Authenticatable.get(session.auth_id)
-        auth.status = "online"
-        auth.last_seen = dt.datetime.utcnow()
+        auth.status = AuthStatus.ONLINE.value
+        auth.last_seen = dt.datetime.now(dt.timezone.utc)
```
Comment (bartvanb) [inline vantage6-server/vantage6/server/websockets.py:348]: Nice that you caught this as well!

**Review by bartvanb — APPROVED** [review]
Looks great! One small style comment

File: vantage6-algorithm-tools/vantage6/algorithm/tools/mock_client.py (hunk 1 above)
Comment (frankcorneliusmartin) [reply vantage6-algorithm-tools/vantage6/algorithm/tools/mock_client.py:3]: No my linter does not complain. But I keep forgetting we agreed on this. Ill solve it.

**Pull request merged**